spyder/app/mainwindow.py

```diff
@@ -2676,6 +2676,11 @@ def report_issue(self, body=None, title=None, open_webpage=False):
                 url.setQuery(query)
                 QDesktopServices.openUrl(url)
 
+    def report_lsp_error(self, message):

```

Comment:
Let's rename this to `report_pyls_error` (because it only applies to the Python server).